### PR TITLE
Improve the Form UI of the Contact Section @Home Page

### LINKED ISSUE #677: Improve the Form UI of the Contact Section @Home Page
## Requirements:

- Re-design the Contact Form

## General PR Guidelines:

- Each pull request should have an appropriate and short title like "Fixed Bug in Upload Page"

- Describe your intended changes in the description section of the pull request (Use bullet points and phrases)

- Refrain from using phrases like "Hi, I am ..", "Please merge me this OPR", "Thank You..", etc. We are only interested in technical parts

- Attach a screenshot/clip of the change(s)

- Make sure to refer to the respective issue in the respective PR using phrases like ``Resolves #issue_number or Closes #issue_number``.

- Please be patient enough. The project maintainers/mentors would review it as per their schedule. Please do not start putting comments like "Please check this" etc.

- Although we support feedback from everyone in all phases of development, it is highly advised not to put any negative comments in other participant's pull requests.

- Always keep a note of the deadline.

### PR DESCRIPTION
**Pull request by samidha28**
Fixes #677 
Changes:
- Modified the background color and position of circles
- Changed the header and placeholder text
- Increased space between input fields for uniformity
- Changed styling of input fields and textarea when having focus
- Changed the styling of send button

Screenshot:
Before:
Screenshot (173)
After:
Screenshot (174)

### CODE REVIEW

File: src/pages/Home/sections/Contact/components/ExperienceForm/ExperienceForm.jsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ function ExperienceForm() {
   return (
     <div id="info" className={styles.info}>
       <div className={styles.general_info}>
-        <h2>Contact us your query or just send a hello.</h2>
+      {/*<h2>Contact us your query or just send a hello.</h2>*/}
```
Comment (iamdarshshah) [inline src/pages/Home/sections/Contact/components/ExperienceForm/ExperienceForm.jsx:13]:
```suggestion
```
Remove this.

**Review by iamdarshshah — CHANGES_REQUESTED** [review]
@samidha28 Placeholder text should not include `?` in it. It should be updated everywhere it has been used.

**Comment by smaranjitghose** [comment]
> Fixes #677
> Changes:
> 
> * Modified the background color and position of circles
> * Changed the header and placeholder text
> * Increased space between input fields for uniformity
> * Changed styling of input fields and textarea when having focus
> * Changed the styling of send button
> 
> Screenshot:
> Before:
> Screenshot (173)
> After:
> Screenshot (174)

Remove the shapes from the background of the form. @anushbhatia could you please assist with the choice of color palette for it as well as the send button

**Comment by samidha28** [comment]
@iamdarshshah I re-requested review but it's not showing for some reason. I'll work on further changes @anushbhatia please provide suggestions .

**Review by anushbhatia — COMMENTED** [review]
1. https://dribbble.com/shots/14126196-Contact-Form-01
2. https://dribbble.com/shots/14139115-Contact-Form-02
3. https://dribbble.com/shots/14595041-Daily-UI-028

**Pull request closed without merging**